busypie/awaiter.py

```diff
@@ -18,6 +18,9 @@ def __init__(self, condition: 'Condition', evaluator_checker: Checker):
     def _validate_condition(self):
         if self._condition.poll_delay > self._condition.wait_time_in_secs:
             raise ValueError('Poll delay should be shorter than maximum wait constraint')
+        if ((hasattr(self._condition, 'at_least') and self._condition.at_least)
```

Comment:
When you don't have "at_least" property?

Reply:
when you are not going to us it, this is the reason for the discussion if we will need any default value

Reply:
Also, how about moving this function to its own scope, so we can test it independently?

Reply:
> Maybe to simplify we can set it to 0 as default right?

im up for it, (to me should be the lowest value like 100 mili or something like this.
but its going to add a lot of changes to in other tests suites

you want me to make the change?

Reply:
FIXED